webpack.config.js

```diff
@@ -0,0 +1,33 @@
+// Load the default @wordpress/scripts config object
+const defaultConfig = require( '@wordpress/scripts/config/webpack.config' );
+
+const legacyAdminFileNames = [
```

Comment:
Can comment that these are legacy because they use jQuery.

Reply:
Yep, I tweaked the naming and added some comments in 5fa94a1c420f13ad09f9dbb565428104786a2592.